shotover-proxy/src/runner.rs

```diff
@@ -64,44 +65,74 @@ impl Default for ConfigOpts {
     }
 }
 
-pub struct Runner {
+pub struct Shotover {
     runtime: Runtime,
     topology: Topology,
     config: Config,
     tracing: TracingState,
 }
 
-impl Runner {
-    pub fn new(params: ConfigOpts) -> Result<Self> {
+impl Shotover {
+    #[allow(clippy::new_without_default)]
```

Comment:
Why no default?

Reply:
Its just a strange thing to want to do (just call new, its shorter!)
And I didnt want to have to commit to having default in our API.